app/javascript/src/js/formValidations.js

```diff
@@ -1,121 +1,39 @@
 /* eslint-disable */
 
-import validate from 'validate.js';
-
-const constraints = {
-  'user[username]': {
-    length: {
-      minimum: 4,
-      maximum: 20,
-    },
-  },
-  'user[email]': {
-    email: true,
-  },
-  'user[password]': {
-    length: {
-      minimum: 6,
-      maximum: 128,
-    },
-  },
-  'user[password_confirmation]': {
-    equality: {
-      attribute: 'user[password]',
-      message: '^The passwords do not match',
-    },
-  },
-  'user[current_password]': {
-    length: {
-      minimum: 6,
-      maximum: 128,
-    },
-  },
-  'user[learning_goal]': {
```

Comment:
The old validations must have handled the settings page validations too. Can we add a validation for the learning goal and apply validations to the [profile](https://github.com/TheOdinProject/theodinproject/blob/main/app/views/devise/registrations/_profile.html.erb), [learning goal](https://github.com/TheOdinProject/theodinproject/blob/main/app/views/devise/registrations/_learning_goal.html.erb) and [password](https://github.com/TheOdinProject/theodinproject/blob/main/app/views/devise/registrations/_password.html.erb) settings page forms too please?